### PR TITLE
Included many sections as tex files

### PR DESCRIPTION
**Pull request by marcopoli**
I included meal info, venue, etc... directly as tex file

Open Issues:
- Daily Timetable for Main Conference
- Workshop program format

### CODE REVIEW

**Review by zhzhang — CHANGES_REQUESTED** [review]
Are the files in `example/handbook` able to be used with the proceedings generation script as well? If not, we will have to duplicate a lot of the files so that both can be generated. Are you able to merge the `example/handbook` with the `example/sigdial` folders so that both can be used together to represent the needs of the whole conference with one input?

File: aclpub2/generate.py [hunk 1]
```diff
@@ -197,6 +217,117 @@ def process_program_proceedings(program):
...
+                    pages.append(current_page)
+                    current_page = []
+                    total_lines = 0
+        pages.append(current_page)
+        entries_by_date[date] = pages
+        current_page = []
+        total_lines = 0
+    return sorted(entries_by_date.items())
+
+
```
[the remark this answers is not in the document]
Comment (marcopoli) [reply aclpub2/generate.py:328]: Yes, the handbook example includes the sidgial files so it is possible to compile both of them without problems.

File: aclpub2/generate.py [hunk 2]
```diff
@@ -56,8 +56,26 @@ def generate(*, path: str, proceedings: bool, handbook: bool, overwrite: bool):
         subprocess.run(["pdflatex", f"-output-directory={build_dir}", str(tex_file)])
 
     if handbook:
+        # Load and preprocess the .yml configuration.
```
[the remark this answers is not in the document]
Comment (marcopoli) [reply aclpub2/generate.py:59]: Sure I Agree with you about this.

File: aclpub2/generate.py [hunk 3]
```diff
@@ -197,6 +217,117 @@ def process_program_proceedings(program):
...
+                    pages.append(current_page)
+                    current_page = []
+                    total_lines = 0
+        pages.append(current_page)
+        entries_by_date[date] = pages
+        current_page = []
+        total_lines = 0
+    return sorted(entries_by_date.items())
+
+def process_program_workshop_handbook(program, max_lines = 35, paper_median_lines = 3, header_lines = 2):
```
[the remark this answers is not in the document]
Comment (marcopoli) [reply aclpub2/generate.py:274]: It is just to keep the functions distinct in case we need to modify one of them.

**Pull request closed without merging**